### DISCUSSION TITLE
How to add a footer in pdf render of a book ?

### DISCUSSION THREAD

**Discussion by jpecquais** [quarto-dev/quarto-cli#4577] [Q&A] [answered]
Hello there,

I'm looking for a solution to add a copyright mention in the footer section of the pdf render of a book. So the footer of every pages should have the page number and the copyright mention.

I have the impression that I should go through a Latex template, which sound like a lot of hassle (especially because I've never done some latex before).

Does anybody here would have any recommendation ?
Thanks !

**Comment by mcanouil** [comment, marked as answer]
See <https://tex.stackexchange.com/questions/130037/header-and-footer-in-scrbook#130039> or any similar thread on the internet.

Then see Quarto template partials: <https://quarto.org/docs/journals/templates.html#latex-partials>.


> I have the impression that I should go through a Latex template, which sound like a lot of hassle (especially because I've never done some latex before).

Not an impression ...
PDF is from LaTeX 🤷‍♂️

**Reply by jpecquais** [reply]
Hi @mcanouil and thanks for your answer !
While I understood that Quarto's pdf are generated with LaTeX, I was surprise not to find a key in the quarto metadata to define footer and header content ;)

But, thanks to your help, I've managed to found a way to understand better the Koma script doc, so everything works fine now !
Thanks again :)
